config.toml

```diff
@@ -34,3 +34,4 @@ home = [ "HTML", "REDIRECTS" ]
 
 [params.versions]
 latest = "1.6"
+dockerImageLatest = "1.6.0"
```

Comment:
Not sure what we need this. We're been using major.minor on the rest of the pages.

Reply:
There seem to be two tags per version in Docker Hub, for example `1.6.0` *and* `1.6`. I'm assuming that the two are equivalent. So we can just use the normal `latest` for this 👍